_posts/using_npkill.md

```diff
@@ -0,0 +1,37 @@
+# Using NPKILL
+
+## What is NPKILL?
+
+npkill is a command line tool that searches for all instances of those clunky and bloated _node_modules/_ folders and then offers you a convientient TUI (terminal user interface) to select and delete them.
+
+## Running NPKILL
+
+NPKILL doesn't have to be installed, as you can just use NPX to run it.
```

Comment:
This statement sounds misleading - doesnt NPX download and install it for NPX usage?

Reply:
addressed